apps/cli/src/commands/LaunchSnack.ts

```diff
@@ -39,7 +40,18 @@ async function launchSnackOnIOSAsync(snackURL: string, deviceId: string, version
   await AppleDevice.openSnackURLAsync(deviceId, snackURL);
 }
 
-export async function getSDKVersionForSnack(snackURL: string): Promise<string | undefined> {
+/** Get SDK version from an EAS Update, or classic updates Snack URL */
+function getSDKVersionForSnackUrl(snackURL: string) {
+  const snack = parseRuntimeUrl(snackURL);
+  if (snack?.sdkVersion) {
+    return `${snack.sdkVersion}.0.0`;
+  }
```

Comment:
For simplicity reasons, this only returns the major version. But we might need to revert that to just return the full SDK version 🤷